.drone.star

```diff
@@ -0,0 +1,170 @@
+def main(ctx):
+    server_versions = [
+        {
+            "value": "daily-master",
+            "tarball": "https://download.owncloud.org/community/owncloud-daily-master.tar.bz2",
+            # 'sha256': '488fc136a8000b42c447e224e69f16f5c31a231eea3038614fdea5e0c2218457',
+        },
+    ]
+
+    client_versions = ["2.5", "2.6"]
+
+    test_suites = [
+        "reshareDir",
+        "scrapeLogFile",
+        "shareDir",
+        "shareFile",
+        "shareGroup",
+        "sharePermissions",
+        "uploadFiles",
+        "basicSync",
+        "concurrentDirRemove",
+        "shareLink",
+        "shareMountInit",
+        "sharePropagationGroups",
+        "sharePropagationInsideGroups",
+        "chunking",
+        "nplusone",
+    ]
+
+    pipelines = []
+
+    for client_version in client_versions:
+        for server_version in server_versions:
+            for test_suite in test_suites:
+                pipelines.append(smashbox(ctx, client_version, server_version, test_suite))
+
+    return pipelines
+
+def smashbox(ctx, client_version, server_version, test_suite):
+    return {
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "%s-%s-%s" % (client_version, test_suite, server_version["value"]),
+        "workspace": {
+            "base": "/var/www",
+        },
+        "platform": {
+            "os": "linux",
+            "arch": "amd64",
+        },
+        "steps": [
+            {
+                "name": "download",
+                "image": "plugins/download",
+                "pull": "always",
+                "settings": {
+                    "username": {
+                        "from_secret": "download_username",
+                    },
+                    "password": {
+                        "from_secret": "download_password",
+                    },
+                    "source": server_version["tarball"],
+                    # "sha256": server_version["sha256"],
+                    "destination": "owncloud.tar.bz2",
+                },
+            },
+            {
+                "name": "extract",
+                "image": "owncloud/ubuntu:latest",
+                "pull": "always",
+                "commands": [
+                    "tar -xjf owncloud.tar.bz2 -C /var/www",
```

Comment:
This will be lost for the next step.

Reply:
Huh?! Why would it? `/var/www` is the workspace. And as you can see in drone, this builds just fine.

Reply:
Oh... you are setting a workspace...

Reply:
Yeah, I basically ported everything as close to the original as possible while porting from 0.8 to 1.0. Please don't make refactor anything here that wasn't better before ;-)